<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CryptorWindow</class>
 <widget class="QMainWindow" name="CryptorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cryptor</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhUppercaseOnly</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="maskLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Маска входных файлов</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="maskText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoFormatting">
     <set>QTextEdit::AutoNone</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="placeholderText">
     <string>Например, *.txt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outputPathButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать папку</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoFormatting">
     <set>QTextEdit::AutoNone</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="placeholderText">
     <string>Например, C:/TopSecret/</string>
    </property>
   </widget>
   <widget class="QLabel" name="pathLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Путь для сохранения файлов</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="strategyNameBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Добавление в конец файла</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Перезапись содержимого</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Счетчик к имени файла</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="strategyLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Если имя файла повторяется</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="onceRadioButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Разовый запуск</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="timerRadioButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Работа по таймеру</string>
    </property>
   </widget>
   <widget class="QLabel" name="planningLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Планирование запуска</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timerLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Опрос файлов (секунды)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="askFileTimerBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="byteGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>491</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Значение 8 байт для модификации</string>
    </property>
    <widget class="QLineEdit" name="byteText_0">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhLatinOnly|Qt::ImhPreferUppercase|Qt::ImhUppercaseOnly</set>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="byteText_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="byteText_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="byteText_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="byteText_4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="byteText_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="byteText_6">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="byteText_7">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="deletionRequiredBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалять входные файлы</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Запустить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoFormatting">
     <set>QTextEdit::AutoNone</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="placeholderText">
     <string>Например, C:/NotSecret/</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Путь к входным файлам</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="inputPathButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать папку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выключить</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>inputText</tabstop>
  <tabstop>inputPathButton</tabstop>
  <tabstop>maskText</tabstop>
  <tabstop>outputText</tabstop>
  <tabstop>outputPathButton</tabstop>
  <tabstop>strategyNameBox</tabstop>
  <tabstop>onceRadioButton</tabstop>
  <tabstop>timerRadioButton</tabstop>
  <tabstop>askFileTimerBox</tabstop>
  <tabstop>byteText_0</tabstop>
  <tabstop>byteText_1</tabstop>
  <tabstop>byteText_2</tabstop>
  <tabstop>byteText_3</tabstop>
  <tabstop>byteText_4</tabstop>
  <tabstop>byteText_5</tabstop>
  <tabstop>byteText_6</tabstop>
  <tabstop>byteText_7</tabstop>
  <tabstop>deletionRequiredBox</tabstop>
  <tabstop>startButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
